index.js

```diff
@@ -14,6 +14,7 @@ const CONFIRMATIONS = parseInt(process.env.CONFIRMATIONS || "3")
 const NODE_URL = process.env.NODE_URL || 'http://litecoind.default.svc.cluster.local:9332'
 const RPC_USERNAME = process.env.RPC_USERNAME || 'rpcuser'
 const RPC_PASSWORD = process.env.RPC_PASSWORD || 'rpcpassword'
+const EXPORT_TIMEOUT_MLS = parseInt(process.env.EXPORT_TIMEOUT_MLS || 1000 * 60 * 15)     // 5 minutes
```

Comment:
The comment here should says '15 minutes' right? But even so may be too low for the utxo chains as every block is on 15 minutes. Maybe the default value should be like 20 minutes.

Reply:
The comment was fixed.
I rechecked k8s-apps in devops and see that there are 3 apps using this image, which as I understand can have different block times. So maybe better solution is to explicitly configure each app instead of rely on default value?
@valo @YordanPavlov

Reply:
For BTC the block time is 10min. Looking at the historical block times, seems like 15min will work 95% of the cases: https://bitinfocharts.com/comparison/bitcoin-confirmationtime.html

Reply:
@apozdeyev You are right, this image is used for Litecoin which has lower times.